Directory.Packages.props

```diff
@@ -0,0 +1,33 @@
+<Project>
+  <PropertyGroup>
```

Comment:
nit: for readability, please add an empty line before and after the property group and item groups. See https://github.com/dotnet/format/blob/main/Directory.Packages.props for an example.

Reply:
Done